01/metallrooch/wordvalue.py

```diff
@@ -0,0 +1,28 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, 'r') as dictionary_file:
+        lst = dictionary_file.read().split('\n')[:-1]
+    return lst
```

Comment:
certainly good, you can omit the temp `lst` variable, using `return dictionary_file ...`